bats/fb-content-katello.bats

```diff
@@ -121,6 +121,12 @@ EOF
   tPackageExists subscription-manager || tPackageInstall subscription-manager
 }
 
+@test "disable puppet agent to prevent checkin from registering host to another org" {
+  if systemctl is-active puppet; then
```

Comment:
Maybe it's nice to clearly show if the service was running or not (same with starting):

```suggestion
  systemctl is-active puppet || skip "Not running"
```